src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Update/_breadcrumb.html.twig

```diff
@@ -3,7 +3,7 @@
 {% set breadcrumbs = [
         { label: 'sylius.ui.administration'|trans, url: path('sylius_admin_dashboard') },
         { label: 'sylius.ui.products'|trans, url: path('sylius_admin_product_index') },
-        { label: app.request.attributes.get('productId') },
+        { label: app.request.attributes.get('productId'), url: path('sylius_admin_product_index') ~ app.request.attributes.get('productId') },
```

Comment:
Please use the proper route for this url: `sylius_admin_product_update`

Reply:
Why? We need create link that it moves us to product index page not product edit page

Reply:
So.. should it moves us to product index page after clicking a **product id**?

And what is more, your implementation moves us to product show page, what you show us in screenschots, so I do not understand your comment..

Reply:
After clicking a product id on a product variant edit page, we should be moved to `sylius_admin_product_update` page